android/app/src/main/java/org/fossasia/openevent/adapters/DayScheduleAdapter.java

```diff
@@ -107,6 +109,17 @@ public void onBindViewHolder(DayScheduleViewHolder holder, int position) {
         }
         holder.slotLocation.setText(currentSession.getMicrolocation().getName());
 
+        Track currentSessionTrack = currentSession.getTrack();
+        holder.slotTrack.setText(currentSessionTrack.getName());
```

Comment:
The bold text works well to show the text to the user if the color of the tag is too light